apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -143,9 +143,13 @@ export default {
     ...mapState(['user']),
 
     currentPanel() {
-      const panel = this.currentSidebarTab.options?.collaboratorsCurrentPanel
+      const tabOptions = this.currentSidebarTab.options
 
-      return panel || PANEL_SHOW
+      if (tabOptions) {
+        return tabOptions.collaboratorsCurrentPanel
```

Comment:
Same as above, is `tabOptions.collaboratorsCurrentPanel` guaranteed to have a value? The logic we had before falls back to `PANEL_SHOW` if the options are not there or if the `collaboratorsCurrentPanel` is empty. New code only performs the first of the two checks.